frontend/scripts/run-android-local.sh

```
@@ -0,0 +1,198 @@
+    echo ""
+    echo "Usage: $0 [env-file] [emulator-name]"
+    echo "Examples:"
+    echo "  $0                 # Uses .env.android (default) and auto-selects emulator"
+    echo "  $0 .env.dev        # Uses .env.dev and auto-selects emulator"
+    echo "  $0 .env.local Pixel_3a_API_34_extension_level_7_arm64-v8a"
+    exit 1
+fi
+
+# Load environment variables from specified file
+echo -e "${GREEN}✅ Loading environment from: ${ENV_FILE}${NC}"
+source "$ENV_FILE"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)